tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -563,7 +563,7 @@ install_kernel_nvidia_gpu() {
 	install_kernel_helper \
 		"assets.kernel.version" \
 		"kernel-nvidia-gpu" \
-		"-g nvidia -u ${kernel_url} -H deb"
+		"-g nvidia -D nvidia -u ${kernel_url} -H deb"
```

Comment:
@zvonkok please let know if it's okay to piggyback DPU driver in gpu kernel, thanks.